packages/cursor/setup.sh

```
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# Imports a GPG key, presets the passphrase, and configures git commit signing.
+# Downloaded and executed by each project's .cursor/scripts/install-gpg.sh.
+# Supports both passphrase-protected and passwordless keys.
+# Extracts git identity (email and name) from the key UID.
+
+set -euo pipefail
+
+# Skip if GPG signing is already fully configured.
+if git config --global user.signingkey &>/dev/null \
+   && [[ "$(git config --global --get commit.gpgsign 2>/dev/null)" == "true" ]]; then
```

Shellcheck (0.11.0)

[info] Consider invoking this command separately to avoid masking its return value (or use '|| true' to ignore).

(SC2312)